railties/lib/rails/secrets.rb

```diff
@@ -30,20 +32,23 @@ def parse(paths, env:)
       end
 
       def generate_key
-        cipher = new_cipher
-        SecureRandom.hex(cipher.key_len)[0, cipher.key_len]
+        cipher = OpenSSL::Cipher.new(CIPHER)
+        key = SecureRandom.random_bytes(cipher.key_len)
+
+        key.unpack('H*').first
```

Comment:
Just a small code style thing, this string should be double-quoted, but don't bother changing only this, I can do it.